lib/spack/spack/test/install.py

```diff
@@ -5,11 +5,28 @@
 
 import os
 import pytest
+import shutil
+
+from llnl.util.filesystem import mkdirp, touch, working_dir
 
 import spack.patch
 import spack.repo
 import spack.store
 from spack.spec import Spec
+from spack.package import _spack_build_envfile, _spack_build_logfile
+
+
+@pytest.fixture
+def mock_log():
```

Comment:
Is this fixture being used? I'm likely missing something but I don't see it being used when skimming through the diff.

Reply:
Not anymore.  Thanks for catching it.